src/2.6.0/wso2apim.js

```diff
@@ -154,20 +158,17 @@ async function isCertUploaded(url, accessToken, certAlias) {
   }
 }
 
-
-
 function constructAPIDef(user, gatewayEnv, apiDef, apiId) {
   try {
-
     // Construct backend-specific parameters
-    var backendBaseUrl, backendType, backendTransport;
+    var backendBaseUrl, backendType;
```

Comment:
I'd stick to either let or const

Reply:
Yup, i wanted this variable scope to be at higher level. in this case we need to set the values conditionally below. So how do you avoid scoping issues without declaring it higher. Any thoughts!

Reply:
var is function level scope, let const are block level scope. In this case I think you'd get away with let because they're at function scope anyway.

Reply:
Perfect, `let` worked! :)